### ISSUE TITLE
Unable to install plugins on localhost

### ISSUE THREAD

**Issue by imfaisalkh** [richtabor/MerlinWP#146]
I'm testing Merlin WP with my theme on my local installation. I'm already using TGM for plugin activation and have included Merlin after TGM (as mentioned in the docs). Now the issue is whenever Merlin tries to install the plugins on localhost I get "Cross-Origin Request Blocked" error in the browser console. Any idea how to resolve this issue?

![image](https://user-images.githubusercontent.com/10553273/49438469-3fcdff80-f7e0-11e8-8585-fae2c81f4757.png)

Also, although plugins are not installed it shows big green tick (at the top) at the end and behaves like plugins are installed successfully.

**Comment by imfaisalkh** [comment]
I think I got the issue. I'm running this installation at following URL: http://localhost:7000/wordpress/fresh-install/

Whereas, Merlin is trying to access TGMPA at following URL:
http://localhost/wordpress/fresh-install/ (without port address)

The port address is added by browserSync for live reloading. I've dropped the port address and now it works fine. Although, I do think any failure in plugin installation needs to be handled, otherwise it creates confusion.

BTW another small issue, in Firefox, the image is not aligned centered on theme on-boarding screen as posted above.
